node/src/Pos/Wallet/Web/Secret.hs

```diff
@@ -34,7 +35,7 @@ instance Buildable WalletUserSecret where
     build WalletUserSecret{..} =
         bprint ("{ root = "%addressF%", set name = "%build%
                 ", wallets = "%pairsF%", accounts = "%pairsF%" }")
-        (makePubKeyAddress $ encToPublic _wusRootKey)
+        (makePubKeyAddress undefined $ encToPublic _wusRootKey)
```

Comment:
I think it's safe to provide `IsBootstrapEra False` instead of `undefined` here, because root key addresses are never actually used to store and send money, and are used only for identifying particular HD address tree. Actually, I think that there's no significant difference between `IsBootstrapEra False` and `IsBootstrapEra True` here, but it's better to make some helper (like `makeRootPubKeyAddress`) to avoid confusion.